### ISSUE TITLE
Style loss: MSE or SSE

### ISSUE THREAD

**Issue by Mona77** [eridgd/AdaIN-TF#6]
Hi, I was wondering why you have calculated `sum of squared` in style loss calculations:

`m_loss = sse(d_mean, s_mean) / batch_size # normalized w.r.t. batch size`

While in torch implementation `MSE` is being used, [this line](https://github.com/eridgd/AdaIN-TF/blob/master/model.py#L121):

`self.mean_criterion = nn.MSECriterion()
self.mean_loss = self.mean_criterion:forward(self.input_mean, self.target_mean)`

**Comment by eridgd** [comment]
It's been too long for me to remember exactly why I implemented things the way I did, but my guess is it's a mistake. The results do still seem to be comparable to the Torch version, but at some point I'll retrain with MSE and see how it affects the output. Good catch!

**Comment by Mona77** [comment]
@eridgd, I confirm that. I trained your network as it is, and the results are very similar to the Torch version. 
Please let me know. Thank you!
